server/index.js

```diff
@@ -7,6 +7,10 @@ import initExpress from './init/express';
 import initRoutes from './init/routes';
 import renderMiddleware from '../app/server';
 
+const webpackDevMiddleware = require('webpack-dev-middleware');
```

Comment:
maybe require these conditionally inside the `isDebug` if statement, since they're only used there?